src/quota_aggregator_impl.cc

```diff
@@ -137,6 +142,18 @@ ::google::protobuf::util::Status QuotaAggregatorImpl::Quota(
   // Aggregate tokens if the cached response is positive
   if (lookup.value()->is_positive_response()) {
```

Comment:
Here even for positive, we need to check last_refresh_time to make sure it is not too old